src/benchmark_scenarios/private_chats.py

```diff
@@ -27,3 +27,22 @@ async def idle_relay():
     logger.info("Shutting down node connections")
     await asyncio.gather(*[node.shutdown() for node in relay_nodes.values()])
     logger.info("Finished idle_relay")
+
+
+async def idle_light():
+    # 1 light node alice
+    # 100 light nodes - friends
+    # friends have accepted contact request with alice, who accepted it
```

Comment:
```suggestion
    # friends have accepted contact request with alice
```
```...who accepted it```
If Alice requested it, she doesn't need to also have accepted it, right?
There's just:
1. Friend request
2. Accept request

Not
1. Friend request
2. Alice accepts
3. Friends accept

Right?